README.md

```diff
@@ -96,7 +96,7 @@ spec:
 ```
 
 ### Deploy the operator directly
-Deploying the operator without the use of OLM requires applying the component Kubernetes resources for the operator directly. These resources are bundled and attached to each StreamsHub Console release. The latest release can be found [here](https://github.com/streamshub/console/releases/latest). The resource file is named `console-operator-x.y.z.yaml` where `x.y.z` is the released version.
+Deploying the operator without the use of OLM requires applying the component Kubernetes resources for the operator directly. These resources are bundled and attached to each StreamsHub Console release. The latest release can be found [here](https://github.com/streamshub/console/releases/latest). The resource file is named `streamshub-console-operator-x.y.z.yaml` where `x.y.z` is the released version.
```

Comment:
I guess it will not matter if we have x.y.z in the file name since its already under the release folder / tag.
It might add some clarity once the file is lets say somewhere on someones local drive and it's just called console-operator.yaml. 
So i am neutral with this one, maybe leaning a bit into removing x.y.z.

Reply:
Nah, lets just remove it 😆 . It looks better that way.
`https://github.com/streamshub/console/releases/download/0.3.3/streamshub-console-operator.yaml`

Reply:
![image](https://github.com/user-attachments/assets/900d3e45-d82a-4a4f-af31-c08412ca9618)